components/compass-runtime-agent/internal/kyma/secrets/strategy/oauth_test.go

```diff
@@ -0,0 +1,245 @@
+package strategy
```

Comment:
We could think of some common library as there is similar code in Application (the only difference is data type).

Reply:
Yes, in different  PR